src/core/referencingfeaturelistmodel.h

```diff
@@ -55,6 +57,18 @@ class ReferencingFeatureListModel : public QAbstractItemModel
     int columnCount( const QModelIndex &parent = QModelIndex() ) const override;
     QVariant data( const QModelIndex &index, int role = Qt::DisplayRole ) const override;
 
+    //! Returns the relation id of the connection to the parent feature with the children in this model
```

Comment:
Actually it set's the relation and not the id, right?

Reply:
It sets the relation via setting the id. Probably you are right what is the overall effect, I will change it.